p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -4876,22 +4877,18 @@ match_kind {
 ~ End P4Example
 
 These identifiers correspond to the P4~14~ match kinds with the same
-names. The semantics of these annotations is actually irrelevant for
-describing the behavior of the P4 abstract machine; how they are used
-influences only the control-plane API and the actual implementation of
-the look-up table. From the point of view of the P4 program, a look-up
-table is an abstract finite map that is given a key and produces as a
-result either an action or a "miss" indication, as described in
-Section [#sec-mau-semantics].
```

Comment:
Earlier we defined the data plane semantics as a function from vectors of bits to vectors of bits. And I think that's the plain English reading of the term.

(Minor nit: as I wrote elsewhere, since `default_action` is no longer mandatory, I don't think it's the case case that given a key you get back an action. You either get a Miss, or a Hit and an action.)

I am worried that some readers will interpret this text as follows: if I take a snapshot of the state populated by the control plane (or use constant entries, as in this example), then the data plane will behave the same (i.e., have the same semantics) no matter what kind of `match_kind` I use with my table. And that's would plainly be false. 

Does this sentence serve an important definitional purpose? If so, then I could be talked into keeping it. If not, I think it's probably better to remove it.